### DISCUSSION TITLE
modifier flag

### DISCUSSION THREAD

**Discussion by iceman-76** [mbround18/valheim-docker#762] [Q&A] [answered]
Hey folks is there a way to pass the modifier flag (or any flag for that matter) in docker-compose or any other way? I have this info:

> World modifiers can also be set individually by using the -modifier flag with following possible arguments and values: 
Combat: veryeasy, easy, hard, veryhard
DeathPenalty: casual, veryeasy, easy, hard,
hardcore
Resources: muchless, less, more, muchmore,
most
Raids: none, muchless, less, more, muchmore
Portals: casual, hard, veryhard

So how would I make sure that the world starts with `-modifier Resources muchmore` - I'm not actually even sure who would you call it.

Thanks

**Comment by iceman-76** [comment]
Or I misunderstood and it would just be` -resources "muchmore"` and in that case it would just be a line under the `environment` section in docker-compose.yml as something like
` - RESOURCES "muchmore"` ?

**Comment by spannerman79** [comment, marked as answer]
```yaml
MODIFIERS: "resources=muchmore"
```

**Comment by iceman-76** [comment]
I swear I looked at the readme... then I went back and I read it - it's there sorry and thank you....
